applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/prj_release.conf

```diff
@@ -15,6 +15,8 @@ CONFIG_CAF_CLICK_DETECTOR=y
 CONFIG_CAF_LEDS=y
 
 CONFIG_DESKTOP_ROLE_HID_DONGLE=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DC
+CONFIG_DESKTOP_DEVICE_PRODUCT="Dongle nRF53 Desktop"
```

Comment:
To be honest I would prefer not to rely on SoC in the Kconfig default. Most of the devices use nRF52 SoC so that is default,
Otherwise this value is assigned explicitly in the configuration. Removing `52` globally could break backwards compatibility.